packages/pds/src/db/migrations/20230605T235529700Z-outgoing-repo-seq.ts

```diff
@@ -0,0 +1,44 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<any>, dialect: Dialect): Promise<void> {
+  if (dialect === 'sqlite') {
+    await db.schema.dropTable('repo_seq').execute()
+    await db.schema
+      .createTable('repo_seq')
+      .addColumn('id', 'integer', (col) => col.autoIncrement().primaryKey())
+      .addColumn('seq', 'integer', (col) => col.unique())
+      .addColumn('did', 'varchar', (col) => col.notNull())
+      .addColumn('eventType', 'varchar', (col) => col.notNull())
+      .addColumn('event', sql`blob`, (col) => col.notNull())
+      .addColumn('invalidated', 'int2', (col) => col.notNull().defaultTo(0))
+      .addColumn('sequencedAt', 'varchar', (col) => col.notNull())
+      .execute()
+  } else {
+    await db.schema.alterTable('repo_seq').renameColumn('seq', 'id').execute()
```

Comment:
We may also want to rename the `seq_pkey` constraint to`id_pkey`.

Reply:
good call 👍 

kysely didn't have a fn for that so I had to hand write it